src/main/java/org/pcollections/ConsPStack.java

```diff
@@ -7,10 +7,7 @@
 package org.pcollections;
 
 import java.io.Serializable;
-import java.util.AbstractSequentialList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.ListIterator;
+import java.util.*;
```

Comment:
Please configure your IDE to do not collapse imports. Import only class which class use

Reply:
✔️ 
Originally I saw `org.pcollections.*` import in tests, so I concluded that star imports are not discouraged.